app/controllers/links_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
     else
       Link
     end
-    @links = scope.where("published_at IS NOT NULL").order(published_at: "DESC")
+    @links = scope.order(published_at: :desc)
```

Comment:
@etagwerker could you take a look at this change please. You originally added the `where` we need to remove it at this stage because we want to show all the links on the index. This also means that links without a `publish_at` will be at the top of the list.

Do you agree with this change?